src/offsetRotate.js

```diff
@@ -0,0 +1,92 @@
+/* global internalScope */
+
+(function () {
+  function isNumeric (number) {
+    return !isNaN(number);
+  }
+
+  function convertToDegrees (angle) {
+    var angleUnitArray = /(deg|grad|rad|turn)$/.exec(angle);
+    if (angleUnitArray === null) {
+      return undefined;
+    }
+
+    var unit = angleUnitArray[0];
+    var angleDegrees = angle.substring(0, angle.length - unit.length);
+    if (!(isNumeric(angleDegrees))) {
+      return undefined;
```

Comment:
What happens when the return undefined code path is hit?
We should have tests for this.